app/src/main/res/layout/exploration_fragment.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
+
+  <androidx.constraintlayout.widget.ConstraintLayout
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <TextView
+      android:id="@+id/dummy_text_view"
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:text="@string/dummy_string"
```

Comment:
Let's hardcode the string rather than introducing a new string variable. Eventually, we will need to support translating strings, and it's easy to lose track of them. If this is a temporary thing, we can just hardcode the string here & change it in a subsequent PR.

Reply:
Makes sense. Will follow this tip from now on.